api-client/src/main/java/com/desk/java/apiclient/service/UserService.java

```diff
@@ -127,26 +128,72 @@
 
     /**
      * Retrieve a list of mobile device settings.
-     * @see <a href="http://dev.desk.com/API/users/#mobile-devices-settings-list">http://dev.desk.com/API/users/#mobile-devices-settings-list</a>
      *
-     * @param userId the user id
+     * @param userId   the user id
      * @param deviceId the device id
      * @return a setting api response
+     * @see <a href="http://dev.desk.com/API/users/#mobile-devices-settings-list">http://dev.desk.com/API/users/#mobile-devices-settings-list</a>
      */
     @GET(USERS_URI + "/{userId}/" + MOBILE_DEVICES_URI + "/{deviceId}/" + SETTINGS_URI)
     Call<ApiResponse<Setting>> getMobileDevicesSettings(@Path("userId") int userId, @Path("deviceId") int deviceId);
 
     /**
      * Update a mobile device setting
-     * @see <a href="http://dev.desk.com/API/users/#mobile-devices-settings-update">http://dev.desk.com/API/users/#mobile-devices-settings-update</a>
      *
-     * @param userId the user id
-     * @param deviceId the device id
+     * @param userId    the user id
+     * @param deviceId  the device id
      * @param settingId the setting id
-     * @param update the setting update body
+     * @param update    the setting update body
      * @return a setting
+     * @see <a href="http://dev.desk.com/API/users/#mobile-devices-settings-update">http://dev.desk.com/API/users/#mobile-devices-settings-update</a>
      */
     @PATCH(USERS_URI + "/{userId}/" + MOBILE_DEVICES_URI + "/{deviceId}/" + SETTINGS_URI + "/{settingId}")
     Call<Setting> updateMobileDeviceSetting(@Path("userId") int userId, @Path("deviceId") int deviceId,
-                                   @Path("settingId") int settingId, @Body SettingUpdate update);
+                                            @Path("settingId") int settingId, @Body SettingUpdate update);
+
+    /**
+     * Retrieves case filters.
```

Comment:
I would maybe add "for the current user" so it's a little more clear that it's getting their filters, not all.